src/Module/AbstractBaseModule.php

```diff
@@ -238,9 +291,16 @@ protected function _initModuleEvents($eventManager, $eventFactory)
      */
     protected function _setupContainer($config, $services)
     {
+        $configData        = $this->_createCompositeContainer([
```

Comment:
Could you describe exactly what is going on in this method?

Reply:
The injected config and the config read from the `config.php` file are merged into a composite container, and a config instance is created using the composite.

Then, the services are read from the `services.php` file and wrapped into a container.

Finally, a composite container is created with the above two results.